UnofficialSteamAuthenticator/AuthenticatorPage.xaml.cs

```diff
@@ -103,46 +105,48 @@ private void LoginBtn_Click(object sender, RoutedEventArgs e)
         private async void LinkResponseReal(AuthenticatorLinker.LinkResult linkResponse)
         {
             bool firstRun = PhoneNumGrid.Visibility == Visibility.Collapsed;
-            Progress.Visibility = ErrorLabel.Visibility = SMSGrid.Visibility = PhoneNumGrid.Visibility = RevocationGrid.Visibility = Visibility.Collapsed;
+            Progress.Visibility = ErrorLabel.Visibility = SmsGrid.Visibility = PhoneNumGrid.Visibility = RevocationGrid.Visibility = Visibility.Collapsed;
```

Comment:
Not automatically. Only two coherent chars are caps (System.IO, for example). Everything above 2 characters is PascalCased. So SMS becomes Sms.